src/analytics.ts

```diff
@@ -82,7 +81,7 @@ export function createAnalyticsInstance(options?: Options) {
     ) => _growthbook?.getFeatureValue(id as string, defaultValue)
     const isFeatureOn = (key: string) => _growthbook?.isOn(key)
     const setUrl = (href: string) => _growthbook?.setUrl(href)
-    const getId = () => _rudderstack?.getUserId() || _rudderstack?.getAnonymousId()
+    const getId = () => _rudderstack?.getUserId()
```

Comment:
Don't we need a fallback here?

Reply:
Actually the issue what the analytics teams faced was that the userId was always returning the anonymous value which is not needed.we need to try checking if this works or not